docs/FAQ/how-to-deploy-agent-in-egg-framework.md

```diff
@@ -0,0 +1,27 @@
+# How to deploy agent in egg framework
+
+## Install `skywalking-egg-require` module
+1. goto egg framework
+2. run the following command
+```shell
+$ npm install skywalking-egg-require --save
+```
+
+## Deploy Skywalking collector
+1. Download releases from Apache official website. [Link](http://skywalking.apache.org/downloads/)
+2. Deploy backend on local. See [collector in standalone mode doc](Deploy-backend-in-standalone-mode.md)
```

Comment:
This should not guide to standalong mode directly, you should guide to document index page, and tell people to read which parts.